src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/IncidentsResourceTest.kt

```diff
@@ -16,10 +19,12 @@ import uk.gov.justice.hmpps.prison.api.model.questionnaire.PrisonerRoleRequest
 import uk.gov.justice.hmpps.prison.api.model.questionnaire.QuestionRequest
 import uk.gov.justice.hmpps.prison.api.model.questionnaire.UpdateIncidentTypeConfigurationRequest
 
+@TestClassOrder(ClassOrderer.OrderAnnotation::class)
```

Comment:
test ordering required as in spring boot upgrade the create / update happened before the retrieve